database.sql

```diff
@@ -1,6 +1,6 @@
 -- ------------------------------------------
 -- Friendica 2022.09-dev (Giant Rhubarb)
--- DB_UPDATE_VERSION 1470
+-- DB_UPDATE_VERSION 1472
```

Comment:
Why did you skip 1471?

Reply:
I had it internally and it was released on my server.